### PR TITLE
Refactor/warning fcm on old config/lp 7724

### PR DESCRIPTION
**Pull request by alexisoyama**

### CODE REVIEW

**Comment by alexisoyama** [comment]
test this please

**Comment by alexisoyama** [comment]
test this please

File: AndroidSDKPush/src/main/java/com/leanplum/LeanplumPushService.java [hunk 1]
```diff
@@ -151,9 +151,22 @@ public static void setGcmSenderIds(String... senderIds) {
...
+   *  For example:
+   *    packagedReleaseCompile 'com.leanplum:Leanplum-core:'
+   *    packagedReleaseCompile 'com.leanplum:Leanplum-fcm:'
+   *    packagedReleaseCompile 'com.leanplum:Leanplum-location:'
    */
+  @Deprecated
   public static void enableFirebase() {
     LeanplumPushService.isFirebaseEnabled = true;
+    Log.e("enableFirebase() is deprecated and FCM is not enabled! " +
+            "SDK has split up into modules and you need to modify your build.gradle. " +
```
Comment (Enuviel) [inline AndroidSDKPush/src/main/java/com/leanplum/LeanplumPushService.java]: has been

**Comment by alexisoyama** [comment]
test this please

**Pull request merged**